cpp/include/cudf/replace.hpp

```diff
@@ -26,6 +26,14 @@ namespace cudf {
  * @file
  */
 
+/**
+ * @brief Policy to fill null/nan values inside
+ *
+ * `FORWARD_FILL` means filling the null
+ * `BACKWARD_FILL` means
+ */
+enum class fillna_policy : bool { FORWARD_FILL, BACKWARD_FILL };
```

Comment:
We should strive to have more descriptive names in libcudf than just reusing Pandas terminology. 
```suggestion
enum class replace_policy : bool {PRECEDING, FOLLOWING};
```